src/components/AddressSearch.js

```diff
@@ -32,6 +53,12 @@ const propTypes = {
 };
 
 const defaultProps = {
+    // ...defaultFieldPropTypes,
+    isFormInput: false,
+    inputID: undefined,
+    shouldSaveDraft: false,
+    onBlur: () => {},
+    errorText: '',
     value: '',
```

Comment:
We should update the default value to `undefined`

Reply:
Done